snippets/snippets.json

```diff
@@ -191,12 +191,36 @@
 		"prefix": "mod",
 		"scope": "v,vlang"
 	},
+	"snippet.public.const": {
+		"body": ["pub const ($1)"],
+		"description": "Code snippet for public constant 'pub const'",
+		"prefix": "pubc",
+		"scope": "v,vlang"
+	},
+	"snippet.public.enum": {
+		"body": ["pub enum ${1:name} {", "\t$0", "}"],
+		"description": "Code snippet for public enum 'pub enum'",
+		"prefix": "pube",
```

Comment:
How about `pe`, `pc`, `ps`, etc? It's shorter and easy enough to remember:

`pc` = <b><ins>p</ins></b>ublic <b><ins>c</ins></b>onst
`pe` = <b><ins>p</ins></b>ublic <b><ins>e</ins></b>num
`ps` = <b><ins>p</ins></b>ublic <b><ins>s</ins></b>truct
… etc

Reply:
good idea !!!

Reply:
so `pf` for `pu fn`

Reply:
I think so. 👍 The current prefix `pub` is a little ambiguous.